dune-project

```diff
@@ -13,13 +13,13 @@
 
 (license MIT)
 
-; (documentation https://url/to/documentation)
+(documentation https://ocaml-multicore.github.io/par_incr/)
 
 (package
  (name par_incr)
  (synopsis "Parallel Self Adjusting Computation")
  (description "Parallel version of the incremental library")
- (depends ocaml dune domainslib (alcotest :with-test) (mdx :with-test))
+ (depends ocaml dune (domainslib :with-test) (alcotest :with-test) (mdx :with-test) (current_incr :with-test) (incremental :with-test))
```

Comment:
I'd recommend giving some version ranges (at least minimum version) for the dependencies.   (I think opam ci might try to build this with OCaml 4.)  I'd also format this with one dependency per line.

Reply:
Ok I'll add version ranges and format it nicely